pkg/image/api/validation/validation.go

```diff
@@ -111,7 +111,7 @@ func ValidateImageStreamTagReference(tagRef api.TagReference, fldPath *field.Pat
 	var errs field.ErrorList
 	if tagRef.From != nil {
 		if len(tagRef.From.Name) == 0 {
-			errs = append(errs, field.Required(fldPath.Child("from", "name"), "name is required"))
+			errs = append(errs, field.Required(fldPath.Child("from", "name"), "name is required for tag reference"))
```

Comment:
Prexisting, but this looks like it adds zero entropy.  Any idea what value the string is adding?